File: gin-vue-admin/web/src/api/ffOperation.js
import service from '@/utils/request'

export const createOperation = (data) => {
  return service({
    url: '/ffProduct/createOperation',
    method: 'post',
    data
  })
}

export const updateOperation = (data) => {
  return service({
    url: '/ffProduct/updateOperation',
    method: 'put',
    data
  })
}

export const deleteOperation = (data) => {
  return service({
    url: '/ffProduct/deleteOperation',
    method: 'delete',
    data
  })
}

export const getOperationList = (params) => {
  return service({
    url: '/ffProduct/getOperationList',
    method: 'get',
    params
  })
}

File: gin-vue-admin/server/router/ffproduction/ff_operation.go
package ffproduction

import (
	"gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type OperationRouter struct{}

func (e *OperationRouter) InitOperationRouter(Router *gin.RouterGroup) {
	operationRouter := Router.Group("ffProduct").Use(middleware.OperationRecord())
	operationRouterWithoutRecord := Router.Group("ffProduct")
	{
		operationRouter.POST("createOperation", operationApi.CreateOperation)
		operationRouter.PUT("updateOperation", operationApi.UpdateOperation)
		operationRouter.DELETE("deleteOperation", operationApi.DeleteOperation)
	}
	{
		operationRouterWithoutRecord.GET("getOperationList", operationApi.GetOperationList)
	}
}
